extensions/wikia/LocalSitemapPage/LocalSitemapPage.setup.php

```diff
@@ -12,4 +12,3 @@
 // hooks
 $wgHooks['ArticleFromTitle'][] = 'LocalSitemapPageHooks::onArticleFromTitle';
 $wgHooks['PageHeaderActionButtonShouldDisplay'][] = 'LocalSitemapPageHooks::onPageHeaderActionButtonShouldDisplay';
-$wgHooks['PageHeaderLanguageSelectorShouldDisplay'][] = 'LocalSitemapPageHooks::onPageHeaderLanguageSelectorShouldDisplay';
```

Comment:
why did you removed this hook? it was on purpose to not display the language on local sitemap page